src/main/java/com/epam/jdi/util/GsonParser.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.jdi.util;
+
+import com.epam.jdi.enteties.MetalsAndColorsInfo;
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.Map;
+
+public class GsonParser {
+
+    public static Object[][] parser(String path) throws IOException {
```

Comment:
Parser is a noun, so we cann't call a method. Method is an action.
parse can be used